docs/AboutUs.md

```diff
@@ -9,15 +9,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ong Wee Marcus
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/lfrostbytee.png" width="200px">
 
 [[homepage](http://www.comp.nus.edu.sg/~damithch)]
```

Comment:
Do you have a homepage you want to link to? This links to the prof's page 😆

Reply:
Yup haha I wanted to actually bring up this issue during the upcoming meeting. Would it be better if I were to make an Issue on the Issue tracker first?

Reply:
Hmm I think the homepage thing is just a sample for us. If we have a homepage then we can link it here, otherwise I think it's fine to remove it. So I don't think you to create an issue? 🤔

Reply:
Sounds like a good suggestion to me. Let me add a new commit that removes the homepage.